quickwit-cli/src/lib.rs

```diff
@@ -391,6 +405,53 @@ fn display_statistics(
     Ok(())
 }
 
+/// ThroughputCalculator is used to calculate throughput on time interval.
+struct ThroughputCalculator {
+    /// clock for tracking the time
+    clock: Instant,
+    /// How long it takes before calculating the throughput.
+    window_size: u64,
```

Comment:
You could make that kind of stuff a const and not have to check for it being 0.
Less code and less trouble.

In practise it will never be configured.